### PR TITLE
AbsToAbsHandler

### PR DESCRIPTION
**Pull request by jonasBoss**
only needs some tests and then should be good to merge

### CODE REVIEW

**Comment by jonasBoss** [comment]
This fixes #441 and implements the AbsToAbsHandler.
From my side it is ready to merge.

File: inputremapper/configs/mapping.py [hunk 1]
```diff
@@ -292,15 +292,24 @@ def validate_output_integrity(cls, values):
...
         output_type = values.get("output_type")
         event_values = [event.value for event in combination]
-        if 0 not in event_values:
-            return values
+        output_symbol = values.get("output_symbol")
 
-        if output_type not in (EV_ABS, EV_REL):
+        if 0 not in event_values and not (
+            output_symbol is not None or output_type == EV_KEY
+        ):
```
Comment (sezanzeb) [inline inputremapper/configs/mapping.py:305]: I think this is equivalent:

```suggestion
        if 0 not in event_values and output_symbol and output_type != EV_KEY:
```

File: inputremapper/injection/mapping_handlers/abs_to_abs_handler.py [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+            return int(max(self._target_absinfo.min, y))
+
+    def _write(self, value: int):
+        """Inject."""
+        try:
+            global_uinputs.write(
+                (*self._output_axis, value), self.mapping.target_uinput
+            )
+        except OverflowError:
+            # screwed up the calculation of mouse movements
```
Comment (sezanzeb) [inline inputremapper/injection/mapping_handlers/abs_to_abs_handler.py:141]: mouse movements?

File: inputremapper/injection/mapping_handlers/abs_to_abs_handler.py [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+
+    def __init__(
+        self,
+        combination: EventCombination,
+        mapping: Mapping,
+        **_,
+    ) -> None:
+        super(AbsToAbsHandler, self).__init__(combination, mapping)
+
+        # find the input event we are supposed to map
```
Comment (sezanzeb) [inline inputremapper/injection/mapping_handlers/abs_to_abs_handler.py:55]: I think I'd add an example to this. I hope I got this right

```suggestion
        # find the input event we are supposed to map. If the input combination is
        # BTN_A + ABS_X + BTN_B, then use the value of ABS_X for the transformation
```

File: inputremapper/injection/mapping_handlers/abs_to_abs_handler.py (hunk 2 above)
Comment (jonasBoss) [reply inputremapper/injection/mapping_handlers/abs_to_abs_handler.py:141]: copy paste error 
```suggestion
            # screwed up the calculation the event value
```

File: inputremapper/configs/mapping.py (hunk 1 above)
Comment (jonasBoss) [reply inputremapper/configs/mapping.py:305]: almost, this works:
```suggestion
        if 0 not in event_values and not output_symbol and output_type != EV_KEY:
```

**Comment by sezanzeb** [comment]
Thanks

**Pull request merged**